### DISCUSSION TITLE
Detect multiple fans to a connector?

### DISCUSSION THREAD

**Discussion by StarwolfWhitefang** [Rem0o/FanControl.Releases#693] [Q&A]
On my Asus ROG Crosshair VIII, I have 3 fans for my AIO radiator connected to the MB CPU fan connector. When I look at the speeds listed in Fan control is says it runs at 80%+ but the rpm seems to be about a third of what the fans are able to run. Any reason for that? Even when I take manual control and ramp it up to 100% the fans barely breaks 550rpm.
What do I do wrong here?

**Comment by Rem0o** [comment]
How do you expect a single RPM channel to be able to read 3 different fans on the same wire?  I suspect the reading is just wrong and should be ignored, your fans should actually spin at the correct rate.

**Comment by StarwolfWhitefang** [comment]
From the noise level and the volume of air I feel floowing out of the top of the cabinet, it is way off from before I tried this software.

**Comment by AtA3301** [comment]
> On my Asus ROG Crosshair VIII, I have 3 fans for my AIO radiator connected to the MB CPU fan connector. When I look at the speeds listed in Fan control is says it runs at 80%+ but the rpm seems to be about a third of what the fans are able to run. Any reason for that? Even when I take manual control and ramp it up to 100% the fans barely breaks 550rpm. What do I do wrong here?

Probably not enough power from a single header to run them all at 100%. Got a fan hub you can try? That takes power from a SATA or Molex connector instead, but still connects to the M/B header to control and monitor speed.

There is no chance of reading how many fans are connected to one header.

**Reply by StarwolfWhitefang** [reply]
Not the problem. Asus AI suite have no problems ramping up the fans to max on one header.
